frontend/src/scenes/ingestion/panels/SuperpowersPanel.tsx

```diff
@@ -0,0 +1,80 @@
+import { LemonSwitch, Link } from '@posthog/lemon-ui'
+import { CardContainer } from 'scenes/ingestion/CardContainer'
+import { useActions } from 'kea'
+import { SupportHeroHog } from 'lib/components/hedgehogs'
+import { useState } from 'react'
+import { teamLogic } from 'scenes/teamLogic'
+import { Tooltip } from 'lib/lemon-ui/Tooltip'
+
+export function SuperpowersPanel(): JSX.Element {
+    const { updateCurrentTeam } = useActions(teamLogic)
+    const [sessionRecordingsChecked, setSessionRecordingsChecked] = useState(true)
+
+    return (
+        <CardContainer
+            nextProps={{ readyToVerify: true }}
+            onContinue={() =>
+                updateCurrentTeam({
+                    session_recording_opt_in: sessionRecordingsChecked,
+                    capture_console_log_opt_in: sessionRecordingsChecked,
+                    capture_performance_opt_in: sessionRecordingsChecked,
+                })
+            }
+        >
+            <div className="flex justify-between items-center -mt-4">
+                <div>
+                    <h1 className="font-extrabold pt-4">Enable your product superpowers</h1>
+                    <p className="m-0">
+                        Collecting events from your app is just the first step toward building great products. PostHog
+                        gives you other superpowers, too, like recording user sessions and automagically capturing
+                        frontend interactions.
+                    </p>
+                </div>
+                <div className="ml-8">
+                    <SupportHeroHog className="w-full h-full" />
+                </div>
+            </div>
+            <div className="my-8">
+                <LemonSwitch
+                    data-attr="opt-in-session-recording-switch"
+                    onChange={(checked) => {
+                        setSessionRecordingsChecked(checked)
+                        updateCurrentTeam({
+                            session_recording_opt_in: checked,
+                            capture_console_log_opt_in: checked,
+                            capture_performance_opt_in: checked,
+                        })
```

Comment:
I don't love this tbh. I think checking the switch should do nothing, and only the continue code should actually trigger it

Reply:
yeah I think you're right, will change